### PR TITLE
Correct the severity of messages.

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch corrects severity of events related to
setting up the webhooks for gluster events api

Signed-off-by: Darshan N <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

File: tendrl/gluster_integration/message/gluster_native_message_handler.py [hunk 1]
```diff
@@ -55,12 +55,16 @@ def _setup_gluster_native_message_reciever(self):
         cmd = cmd_utils.Command('gluster-eventsapi webhook-add %s' % url)
         out, err, rc = cmd.run()
         if rc != 0:
+            severity = "info" if err == "Webhook already exists" else "error"
```
Comment (r0h4n) [inline tendrl/gluster_integration/message/gluster_native_message_handler.py]: Shouldnt this be like below?

`severity = "info" if "Webhook already exists" in err else "error`

Comment (nnDarshan) [reply tendrl/gluster_integration/message/gluster_native_message_handler.py]: ack

**Comment by nnDarshan** [comment]
@r0h4n updated as suggested. Pls review

**Pull request merged**